CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
 
+# 0.24.0 - 2020-09-10
+
+- [Remove](https://github.com/rust-bitcoin/rust-bitcoin/pull/385) the `BitcoinHash` trait
+- [Introduce `SigHashCache` structure](https://github.com/rust-bitcoin/rust-bitcoin/pull/390) to replace `SighashComponents` and support all sighash modes
+- [Add](https://github.com/rust-bitcoin/rust-bitcoin/pull/416) `Transaction::get_size` method
+- [Export](https://github.com/rust-bitcoin/rust-bitcoin/pull/412) `util::amount::Denomination`
+- [Add](https://github.com/rust-bitcoin/rust-bitcoin/pull/417) `Block::get_size` and `Block::get_weight` methods
+- [Add](https://github.com/rust-bitcoin/rust-bitcoin/pull/415) `MerkleBlock::from_header_txids`
+- [Add](https://github.com/rust-bitcoin/rust-bitcoin/pull/429) `BlockHeader::u256_from_compact_target`
+- [Add](https://github.com/rust-bitcoin/rust-bitcoin/pull/448) `feefilter` network message
+- [Cleanup/replace](https://github.com/rust-bitcoin/rust-bitcoin/pull/397) `Script::Instructions` iterator API
+- [Disallow uncompressed pubkeys in witness address generation](https://github.com/rust-bitcoin/rust-bitcoin/pull/428)
+- [Deprecate](https://github.com/rust-bitcoin/rust-bitcoin/pull/451) `util::contracthash` module
+
```

Comment:
```suggestion
- [Add](https://github.com/rust-bitcoin/rust-bitcoin/pull/435) modulo division operation for `Uint128` and `Uint256`
- [Add](https://github.com/rust-bitcoin/rust-bitcoin/pull/436) `slice_to_u64_be` endian conversion method
- [Add](https://github.com/rust-bitcoin/rust-bitcoin/pull/463) `Clone` to `RawNetworkMessage`

```

Reply:
- [Remove](https://github.com/rust-bitcoin/rust-bitcoin/pull/385) `BitcoinHash` trait
- [Add](https://github.com/rust-bitcoin/rust-bitcoin/pull/385) `Block::block_hash()` and `BlockHeader::block_hash()` methods

Reply:
Sure

Reply:
l'll give you the new methods, but I don't think new `#derive`s are worth including in the changelog